### ISSUE TITLE
Is blamod dead?

### ISSUE THREAD

**Issue by IamarockIamanisland** [ntrf/blamod#31]
Hello, I am so sorry for my poor etiquette: Github doesn't have a way of commenting outside of this. I notices that updates stopped for this mod abruptly, and the creator himself has had sparse activity here. Is Blamod dead or just ceased in development right now?

**Comment by ntrf** [comment]
tl;dr: Nope. But Blamod Reborn will be updated later, probably after June 2020.

I'm the only developer of this mod, but BR is not my only project. Due to some legal issues i how spend most of my time with another project. IDK how long it will take, but it's what has to be done. It does hurt Blamod Reborn, but hopefully i'll figure it out by the summer.

Another aspect is that Valve suddenly decided to publish updates for their Source engine games and SDKs. They've fixed some performance problems with "steampipe" HL2 and now BR works much worse then the official game. BR depends on Singleplayer SDK 2013, but that branch was not updated. This means that BR will have to be ported to Multiplayer branch, which most mods are using anyway. As you can imagine -- that's going to take some time. This is my top priority right now... after i'm done with more urgent tasks.

And BTW -- if you have any further questions, you can find me in Sourceruns.org discord. From there you can either ping me in hl2-mods channel or send me a message directly. I don't want to make any dedicated groups, chats and etc before beta release.

**Comment by IamarockIamanisland** [comment]
One more thing, the installation guide seems to be outdated, as when download Blamod, it has a master folder instead of just the folder I need to move to the sourcemod folder.

**Comment by ntrf** [comment]
You've downloaded a source code package. Github isn't exactly friendly to non-programmers. What you need is a release download: https://github.com/ntrf/blamod/releases/tag/v2.0-alpha14
